### PR TITLE
[FA] First draft of Chapter2/Page2

### PR DESCRIPTION
**Pull request by jowharshamshiri**
Added chapter 2/page2 
updated _toctree.yml
Additions to the glossary
Created TRANSLATING.txt

Related to https://github.com/huggingface/course/issues/50

### CODE REVIEW

**Comment by jowharshamshiri** [comment]
Hi @lewtun I have a question about this passage in the original text(CH2/P2):

``
The vector output by the Transformer module is usually large. It generally has three dimensions:

- **Batch size**: The number of sequences processed at a time (2 in our example).
- **Sequence length**: The length of the numerical representation of the sequence (16 in our example).
- **Hidden size**: The vector dimension of each model input.

It is said to be "high dimensional" because of the last value. The hidden size can be very large (768 is common for smaller models, and in larger models this can reach 3072 or more).
``

Reading this and trying to translate it faithfully, I was confused as to what the original text meant. Especially since earlier in the page you talk about the difference btw scalars and vectors and matrices in math, and here switch to talking about NumPy vectors. I have the Persian equivalent of this passage in my translation, it has a few subtle differences:

``
The vector output by the Transformer module is usually large. Information about these vectors is available in the `shape` attribute that generally has three dimensions:

- **Batch size**: The number of sequences processed at a time (2 in our example).
- **Sequence length**: The length of the numerical representation of the sequence (16 in our example).
- **Hidden size**: The vector dimension of each model input.

The hidden size can be very large (768 is common for smaller models, and in larger models this can reach 3072 or more) and that's why the output is said to be "high dimensional".
``

Does this make sense to you? or should I switch back?

**Comment by lewtun** [comment]
Thanks for adding this new section @jowharshamshiri 🤗 !

Regarding your question, I think we want to avoid mentioning `shape` since that's a PyTorch / NumPy specific attribute. To be precise, we could use "tensor" and say 

> the output from the Transformer module is a tensor that typically has three dimensions:

Would the translation of that sentence make more sense in Persian?

File: chapters/fa/TRANSLATING.txt [hunk 1]
```diff
@@ -0,0 +1,67 @@
+1. Use this style guide for Persian. https://apll.ir/wp-content/uploads/2018/10/D-1394.pdf
```
Comment (lewtun) [inline chapters/fa/TRANSLATING.txt:1]: Great job on providing these detailed instructions!

**Review by lewtun — APPROVED** [review]

**Comment by jowharshamshiri** [comment]
> 



> Thanks for adding this new section @jowharshamshiri hugs !
> 
> Regarding your question, I think we want to avoid mentioning `shape` since that's a PyTorch / NumPy specific attribute. To be precise, we could use "tensor" and say
> 
> > the output from the Transformer module is a tensor that typically has three dimensions:
> 
> Would the translation of that sentence make more sense in Persian?

Yes it works, thanks for the clarification.

**Comment by jowharshamshiri** [comment]
> Great, merging this and will deploy on hf.co later today - thanks again for your great work @jowharshamshiri hugs !

Far too kind. Thank you for writing this material.

**Pull request merged**